lib/Map/TableColumn.js

```diff
@@ -164,7 +164,9 @@ var TableColumn = function(name, values, options) {
 
     this._formattedValues = getFormattedValues(this);
 
-    knockout.track(this, ['_type']);  // so that TableStructure can change columnsByType if type changes.
+    // Track _type so that TableStructure can change columnsByType if type changes.
+    // Track _values so that charts can update live with new data.
+    knockout.track(this, ['_type', '_values']);
```

Comment:
Did you by any chance check CSV performance before/after this change?  There's at least a possibility that it could add a lot of overhead.